package.json

```diff
@@ -15,17 +16,18 @@
     "shortcut"
   ],
   "devDependencies": {
-    "@types/node": "^14.0.14",
+    "@codemirror/autocomplete": "^6.16.0",
+    "@codemirror/state": "^6.4.1",
+    "@codemirror/view": "^6.26.3",
+    "@types/node": "^18.7.13",
     "chalk": "^4.1.0",
-    "copy-webpack-plugin": "^6.1.0",
-    "fs-extra": "^9.0.1",
-    "glob": "^7.1.6",
-    "on-build-webpack": "^0.1.0",
-    "tar": "^6.0.5",
-    "ts-loader": "^7.0.5",
-    "typescript": "^3.9.3",
-    "webpack": "^4.43.0",
-    "webpack-cli": "^3.3.11",
-    "yargs": "^16.2.0"
+    "copy-webpack-plugin": "^11.0.0",
+    "fs-extra": "^10.1.0",
+    "glob": "^8.0.3",
+    "tar": "^6.1.11",
+    "ts-loader": "^9.3.1",
+    "typescript": "^4.8.2",
+    "webpack": "^5.74.0",
+    "webpack-cli": "^4.10.0"
```

Comment:
The new dependency versions in this file were taken from the [upstream `package.json` template](https://github.com/laurent22/joplin/blob/dev/packages/generator-joplin/generators/app/templates/package_TEMPLATE.json).